Sources/NIOSSL/SSLErrors.swift

```diff
@@ -18,20 +18,21 @@ import NIOCore
 /// Wraps a single error from BoringSSL.
 public struct BoringSSLInternalError: Equatable, CustomStringConvertible, NIOSendable {
     private enum Backing: Hashable {
-        case boringSSLErrorCode(UInt32)
+        case boringSSLErrorInfo(UInt32, String, UInt)
```

Comment:
Should these be ignored from the `Equatable`/`Hashable` conformance @Lukasa?

Reply:
I'm inclined to say no, though it's an interesting question.

Reply:
Yup! I lean ever-so-slightly towards not including them as the underlying error is still the same regardless of where it was thrown.

Reply:
We could also expose the error code (wrapped in a struct) if we need to compare two errors just base on the error code.